test/CMakeLists.txt

```diff
@@ -363,7 +363,7 @@ celeritas_add_test(celeritas/random/distribution/UniformRealDistribution.test.cc
 if(CELERITAS_USE_CUDA)
   celeritas_add_test(celeritas/random/curand/CurandPerformance.test.cc GPU
     SOURCES celeritas/random/curand/CurandPerformance.test.cu
-    LINK_LIBRARIES CUDA::cudart
+    LINK_LIBRARIES Celeritas::DeviceToolkit
```

Comment:
Is this redundant? (i.e. is it not already coming from libCeleritas?)

Reply:
No it's only coming from libceleritas if CURAND is the generator: otherwise the device toolkit linking should be private